cypress/integration/instance/InstanceCreation.ts

```diff
@@ -60,18 +60,18 @@ describe("the 'Create a SE instance' Modal", () => {
           "0 of 3 steps completed"
         );
         progressStepsStatuses(SEInstanceStatus.ACCEPTED);
-        cy.ouiaId("steps-count", "QE/StackItem", { timeout: 120000 }).should(
```

Comment:
@kelvah do you run this test with dev environment? I think that a creation of a new real bridge takes about 4 minutes. 
https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/integration-tests/src/test/resources/features/bridge.feature#L7